doc/05-Authentication.md

```diff
@@ -171,9 +171,9 @@ Option                   | Description
 -------------------------|-----------------------------------------------
 backend                  | **Required.** Specifies the backend type. Can be set to `ldap`, `msldap`.
 resource                 | **Required.** The name of the LDAP resource defined in [resources.ini](04-Resources.md#resources).
-user\_class              | **Optional.** LDAP user class. Defaults to `user`.
-user\_name\_attribute    | **Optional.** LDAP attribute which contains the username. Defaults to `sAMAccountName` with `msldap` and `uid` with `ldap`.
+base_dn                  | **Optional.** LDAP base dn for groups.
 group\_class             | **Optional.** LDAP group class. Defaults to `group`.
+group\_member\_attribute | **Optional.** LDAP attribute which contains the username. Defaults to `sAMAccountName` with `msldap` and `member` with `ldap`.
 group\_name\_attribute   | **Optional.** LDAP attribute which contains the groupname. Defaults to `sAMAccountName` with `msldap` and `gid` with `ldap`.
 group\_filter            | **Optional.** LDAP group search filter. Requires `group_class` and `group_name_attribute`.
 nested\_group\_search    | **Optional.** Enable nested group search in Active Directory based on the user. Defaults to `0`. Only available with `backend` type `msldap`.
```

Comment:
Additionally, please add the missing options `user_base_dn` and `domain` as well. In case of `domain` please refer to the `Domain-aware Authentication` section below.